.env

```diff
@@ -0,0 +1,22 @@
+INFLUXDB_VERSION=1.3.2-alpine
+INFLUXDB_INIT_VERSION=latest
+MYSQL_VERSION=5.5
+MYSQL_INIT_VERSION=1.5.1
+MEMCACHED_VERSION=1.5.0-alpine
+CADVISOR_VERSION=v0.26.1
+ZOOKEEPER_VERSION=3.4
+
+MON_SIDECAR_VERSION=1.0.0
+MON_KEYSTONE_VERSION=1.1.1
+MON_KAFKA_VERSION=0.9.0.1-2.11-1.1.0
+MON_KAFKA_INIT_VERSION=0.0.1
+MON_ALARMS_VERSION=1.1.0
+MON_GRAFANA_VERSION=4.0.0-1.1.1
+MON_GRAFANA_INIT_VERSION=1.1.0
+
+MON_API_VERSION=master
```

Comment:
We can always go with the approach that basically allows to override the versions (in any way that it is possible) but does not enforce them in anyway. For example, instead of having

**docker-compose.yml:**
```yml
monasca-persister:                                                                                              
  image: monasca/persister:${MON_PERSISTER_VERSION}
```
**.env**:
```sh
MON_PERSISTER_VERSION=latest
```

we could have

**docker-compose.yml:**
```yml
monasca-persister:                                                                                              
  image: monasca/persister:${MON_PERSISTER_VERSION:-latest}
```

and just document the possibility of overriding those _environmental variables_ with the reference to official docs. 

Still that is somehow not the way that would allow to freeze versions between, for example, each OpenStack release, as there is no file that override those versions. So I think I would go with **Option 1**. It seems a bit more explicit, which by all means I favor, as there is less room for error because everything is strictly specified.  The fact that CI would have to periodically run, I don't think this is too much of a problem. There's an option of having jobs scheduled (as in cron) to run periodically. Based on the [docs](https://docs.travis-ci.com/user/cron-jobs/#Detecting-Builds-Triggered-by-Cron), the bot could detect that the jobs is periodic and basically do what you just describe above.

Ok, I will rebase the PR and use explicit masters of each image, where it is possible.